### PR TITLE
typography

### PR DESCRIPTION
**Pull request by vellip**
based on #11 
Add type sizes and basic heading styles and make links bold instead of underlined

### CODE REVIEW

File: liqd_product/assets/scss/_base.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+h6,
+.h6 {
+    font-size: $font-size-xs;
+}
+
+a {
+    color: $link-color;
+    text-decoration: none;
+    font-weight: 600;
```
Comment (xi) [inline liqd_product/assets/scss/_base.scss]: I think this is an awful idea.

File: liqd_product/assets/scss/_variables.scss [hunk 2]
```diff
@@ -48,3 +49,5 @@ $hero-height-mobile: 18.4em;
 @include grid-set('gutter', $padding);
 $planifolia-contrast-dark-default: $text-color;
 $planifolia-contrast-light-default: $text-color-inverted;
+
+$FontPathSourceSansPro: '~npm-font-source-sans-pro/fonts';
```
Comment (xi) [inline liqd_product/assets/scss/_variables.scss]: Where is this used?

File: liqd_product/assets/scss/_variables.scss [hunk 3]
```diff
@@ -30,11 +30,12 @@ $feedback-color-consideration: #ffae00;
 $spacer: 1em;
 $padding: 1rem;
 
-$font-family-sans-serif: "Arial", "Helvetica", sans-serif;
+$font-family-sans-serif: "Source Sans Pro", "Arial", "Helvetica", sans-serif;
```
Comment (xi) [inline liqd_product/assets/scss/_variables.scss]: What is the purpose of Arial and Helvetica here? I get the feeling we are just dragging them along.

File: liqd_product/assets/scss/_variables.scss (hunk 2 above)
Comment (vellip) [inline liqd_product/assets/scss/_variables.scss]: it's documented here: https://github.com/dasrick/npm-font-source-sans-pro

**Pull request merged**